django/db/models/fields/related.py

```diff
@@ -1656,8 +1664,6 @@ def deconstruct(self):
             kwargs['related_name'] = self.remote_field.related_name
         if self.remote_field.related_query_name is not None:
             kwargs['related_query_name'] = self.remote_field.related_query_name
-        if self.remote_field.on_delete != CASCADE:
-            kwargs['on_delete'] = self.remote_field.on_delete
```

Comment:
I may be missing something, but it seems to me that this line should not be removed, it should be made unconditional.